tests/channel_internal_ut/channel_internal_ut.c

```diff
@@ -26,6 +26,8 @@
 #include "c_pal/srw_lock.h"
 #include "c_pal/threadpool.h"
 #include "c_pal/thandle_log_context_handle.h"
+#include "c_pal/sm.h"
+#include "c_pal/log_critical_and_terminate.h"
```

Comment:
>log_critical_and_terminate [](http://example.com/codeflow?start=16&length=26)

not needed

Reply:
We still have the LogCriticalAndTerminate from SRS_CHANNEL_INTERNAL_43_148